Makefile

```diff
@@ -0,0 +1,10 @@
+clean:
+	rm -fr venv lambda.zip ami-built-lambda.zip
+
+virtualenv:
+	virtualenv venv --python=python2.7
+	venv/bin/pip install -r lambda/requirements.txt
+
+zip: clean virtualenv
+	zip lambda.zip lambda/sign_xpi.py lambda/__init__.py
```

Comment:
I would not recommand naming the package lambda because it is a reserved keyword in Python and you will have to rename it for Python 3.6. What about `aws_lambda` for instance?